docs/ref/migration-operations.txt

```diff
@@ -278,6 +278,16 @@ the model with ``model_name``.
 
 Removes the constraint named ``name`` from the model with ``model_name``.
 
+``AlterConstraint``
+-------------------
+
+.. versionadded:: 5.2
+
+.. class:: AlterConstraint(model_name, name, constraint)
+
+Alter the constraint named ``name`` of the model with ``model_name`` with the
```

Comment:
Maybe `Alters` to be consistent with this page:
```suggestion
Alters the constraint named ``name`` of the model with ``model_name`` with the
```

Reply:
Thank you :+1: